### PR TITLE
about

### PR DESCRIPTION
**Pull request by Mrichk0**
це about

### CODE REVIEW

File: src/partials/about.html [hunk 1]
```diff
@@ -1,5 +1,54 @@
-<section class="about">
-  <span class="about__pre-title">tradition and love </span>
-  <h2 class="about__title">how it’s made?</h2>
+<section class="about" id="How">
```
Comment (VovaRubakha) [inline src/partials/about.html]: Change How to low-case (how)

File: src/partials/about.html [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+            srcset="./images/tablet/cow-768@1x.png 1x, ./images/tablet/cow-768@2x.png 2x"
+            media="(min-width: 768px)"
+          />
+          <source srcset="./images/mobile/cow-320@1x.png 1x, ./images/mobile/cow-320@2x.png 2x" />
 
+          <img src="./images/desktop/cow-1200@1x.png" alt="cow" />
+          <!-- width="516" height="516"  -->
+        </picture>
+      </div>
+      <div class="about__blok">
```
Comment (VovaRubakha) [inline src/partials/about.html]: change "blok" to "block"

File: src/sass/layouts/_about.scss [hunk 3]
```diff
@@ -1,3 +1,234 @@
...
+    margin-bottom: 0;
+  }
+  @media screen and (min-width: 1280px) {
+    width: 550px;
+    height: 550px;
+    // padding-left: 114px;
+    // padding-right: 55px;
+  }
+}
+.about__blok {
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: change "blok" to "block"

File: src/partials/about.html [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+          Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus
+          lorem eget, egestas eros. Maecenas eleifend erat at justo fringilla.
+        </p>
+        <p class="about__text--second">
+          Curabitur lacinia enim at ex blandit, vel pellentesque odio elementum. Mauris rhoncus orci
+          in imperdiet placerat. Vestibulum euismod nisl suscipit ligula volutpat, a feugiat urna
+          maximus. Cras massa nibh, tincidunt. <br /><br />
+          Aliquam erat volutpat. Aenean accumsan.
+        </p>
+        <button class="about__botton" type="button">
```
Comment (VovaRubakha) [inline src/partials/about.html]: botton to button

File: src/partials/about.html [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
+          Read more<svg
+            class="about__botton__icon"
```
Comment (VovaRubakha) [inline src/partials/about.html]: botton to button

File: src/sass/layouts/_about.scss [hunk 6]
```diff
@@ -1,3 +1,234 @@
...
+    letter-spacing: 0.04em;
+    margin-bottom: 62px;
+  }
+  @media screen and (min-width: 1280px) {
+    font-size: 14px;
+    // padding-right: 117px;
+    margin-bottom: 45px;
+  }
+}
+.about__botton {
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: botton to button

File: src/sass/layouts/_about.scss [hunk 7]
```diff
@@ -1,3 +1,234 @@
...
+
+  justify-content: center;
+  align-items: center;
+  color: #d41443;
+  cursor: pointer;
+  &:hover {
+    background-color: #ecebeb;
+  }
+}
+.about__botton__icon {
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: botton to button

File: src/sass/layouts/_about.scss [hunk 8]
```diff
@@ -1,3 +1,234 @@
 .about {
-  background: #FFF5F6;
+  background: #fff5f6;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 9]
```diff
@@ -1,3 +1,234 @@
...
+.about__lable {
+  display: block;
+  font-family: Titan One;
+  font-size: 18px;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+
+  margin-bottom: 15px;
+  color: #ffa5ba;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 10]
```diff
@@ -1,3 +1,234 @@
...
+      background-image: url('../images/desktop/sectionbg1200@2x.png') no-repeat center 34%;
+    }
+  }
+}
+
+.about__lable {
+  display: block;
+  font-family: Titan One;
+  font-size: 18px;
+  line-height: 21px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: 21px change to 1.17

File: src/sass/layouts/_about.scss [hunk 11]
```diff
@@ -1,3 +1,234 @@
...
+  text-align: center;
+  text-transform: uppercase;
+
+  margin-bottom: 15px;
+  color: #ffa5ba;
+
+  @media screen and (min-width: 768px) {
+    font-size: 26px;
+    line-height: 30px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: 30px change to 1.15

File: src/sass/layouts/_about.scss [hunk 12]
```diff
@@ -1,3 +1,234 @@
...
+  }
+  @media screen and (min-width: 1280px) {
+    font-size: 30px;
+    line-height: 34px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: 34px change to 1.13

File: src/sass/layouts/_about.scss [hunk 13]
```diff
@@ -1,3 +1,234 @@
...
+  }
+}
+.about__title {
+  font-family: Titan One;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 34px;
+  line-height: 39px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: 39px change to 1.15

File: src/sass/layouts/_about.scss [hunk 14]
```diff
@@ -1,3 +1,234 @@
...
+  text-align: center;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  margin-bottom: 26px;
+  color: #d41443;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 15]
```diff
@@ -1,3 +1,234 @@
...
+  @media screen and (min-width: 768px) {
+    font-size: 48px;
+    line-height: 55px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: 51px change to 1.14

File: src/sass/layouts/_about.scss [hunk 16]
```diff
@@ -1,3 +1,234 @@
...
+  }
+  @media screen and (min-width: 1280px) {
+    font-size: 58px;
+    line-height: 66px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: 66px change to 1.14

File: src/sass/layouts/_about.scss [hunk 17]
```diff
@@ -1,3 +1,234 @@
...
+.about__text {
+  display: block;
+  font-family: DM Sans;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1.62;
+  text-align: left;
+  margin-bottom: 30px;
+
+  color: #000000;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 18]
```diff
@@ -1,3 +1,234 @@
...
+}
+.about__text--second {
+  display: block;
+  font-family: DM Sans;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1.62;
+  text-align: left;
+  margin-bottom: 17px;
+  color: #907e82;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 19]
```diff
@@ -1,3 +1,234 @@
...
+    font-size: 14px;
+    // padding-right: 117px;
+    margin-bottom: 45px;
+  }
+}
+.about__botton {
+  display: inline-flex;
+  width: 176px;
+  height: 44px;
+  background: #ffffff;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 20]
```diff
@@ -1,3 +1,234 @@
...
+  border: none;
+
+  font-family: DM Sans;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 18px;
+
+  justify-content: center;
+  align-items: center;
+  color: #d41443;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: var

File: src/sass/layouts/_about.scss [hunk 21]
```diff
@@ -1,3 +1,234 @@
...
+  cursor: pointer;
+  &:hover {
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss:199]: add focus

File: src/sass/layouts/_about.scss [hunk 22]
```diff
@@ -1,3 +1,234 @@
...
+    background-color: #ecebeb;
```
Comment (VovaRubakha) [inline src/sass/layouts/_about.scss]: create and add var "--accent-alt: #ecebeb;"

**Review by VovaRubakha — CHANGES_REQUESTED** [review]

**Comment by Mrichk0** [comment]
@VovaRubakha done, comments addressed

**Review by VovaRubakha — APPROVED** [review]

**Pull request merged**